pyproject.toml

```diff
@@ -0,0 +1,10 @@
+[tool.black]
+line_length = 130
```

Comment:
We should add `target-version = ['py37']` for consistent commas in function declarations.

Reply:
Agreed. I was yesterday concerned that that would throw off pytype, but it made little difference when using `py34` vs `py37`. I think the issue was something else.